src/main/java/org/phoebus/channelfinder/ChannelRepository.java

```diff
@@ -412,11 +412,17 @@ public void deleteAll() {
      */
     public SearchResult search(MultiValueMap<String, String> searchParameters) {
         BuiltQuery builtQuery = getBuiltQuery(searchParameters);
+        Integer finalSize = builtQuery.size;
+        Integer finalFrom = builtQuery.from;
+
+        if(builtQuery.size + builtQuery.from >= esService.getES_MAX_RESULT_WINDOW_SIZE()) {
+            String message = MessageFormat.format(TextUtil.SEARCH_FAILED_CAUSE,
+                    searchParameters,
+                    "Max search window exceeded, use the " + CFResourceDescriptors.SCROLL_RESOURCE_URI + " api.");
```

Comment:
Could you add a test case to ChannelRepositoryIT that checks that this is returned?

Reply:
I am working in the tests for this...
I am having the same issues that the github CI build is having with respect to running the IT tests.
I think I have an issue ticket... I am trying to run the tests from intellij using 
`failsafe:integration-test -DskipITs=false`

Reply:
I usually use the full method from the github actions.

`mvn clean install test-compile failsafe:integration-test failsafe:verify --batch-mode --fail-at-end -Djacoco.skip=true -DskipITs=false -DskipITCoverage=true`

 I keep meaning to make a PR to simplify it....

Reply:
I think this should be < not <=

Reply:
Sorry, 
Where should it be `<`
Currently, there is just one comparison I see size+from >= window size

Reply:
Sorry. Yes. That comparison should be >

Reply:
Great!! and you are correct, `>` should be used to determine when the elastic query will fail